frontend/.eslintrc.cjs

```diff
@@ -0,0 +1,109 @@
+module.exports = {
+  extends: [
+    'react-app',
+    'airbnb',
+    'plugin:jsx-a11y/recommended',
+    'prettier',
+    'plugin:react/jsx-runtime',
+  ],
+  parser: '@typescript-eslint/parser',
+  plugins: ['jsx-a11y', 'fp', 'prettier', 'import', 'react-refresh'],
+  rules: {
+    'react-refresh/only-export-components': 'error',
+    // Allow JSX within .js files
+    'react/jsx-filename-extension': [
+      1,
+      { extensions: ['.js', '.jsx', '.ts', '.tsx'] },
+    ],
+    // TODO: remove these
+    'no-unused-variable': 'off',
+    '@typescript-eslint/no-unused-vars': 'off',
```

Comment:
I'd personally love for these to _warn_ in development and _error_ in git hooks and CI. I find that full errors for these in development create unhelpful hurdles that slow down the work. However, I do still want our git hooks and CI to fail for these rules